utils/build_swift/build_swift/driver_arguments.py

```diff
@@ -525,8 +525,9 @@ def create_argument_parser():
     option('--stdlib-deployment-targets', store,
            type=argparse.ShellSplitType(),
            default=None,
-           help='list of targets to compile or cross-compile the Swift '
-                'standard library for. %(default)s by default.')
+           help='The targets to compile or cross-compile the Swift standard '
+                'library for. %(default)s by default. Comma separated list: '
+           + ' '.join(StdlibDeploymentTarget.get_target_names()))
```

Comment:
You shouldn't need this `+` here. Just indent the statement to keep it inline with the ones above and try running `python utils/python_lint.py` and `python -m unittest discover -s utils/build_swift -t utils/` to make sure the tests pass. Likely you need to add a new test case to `utils/build_swift/tests/expected_options.py`.

Reply:
Thanks. It is nicer to look at with a trailing `.format` instead of concatenation.